packages/terra-clinical-item-view/src/ItemView.module.scss

```diff
@@ -62,7 +62,6 @@
 
   .content {
     align-items: flex-start;
-    color: var(--terra-clinical-item-view-content-color, rgba(28, 31, 33, 0.65));
```

Comment:
why was this variable removed ? if we are removing the variable then it might have impact on themes which are still using variable `--terra-clinical-item-view-content-color`